src/main/java/StringManipulation.java

```diff
@@ -1,32 +1,99 @@
-public class StringManipulation implements StringManipulationInterface {
+// Sean Michael CS410
+
+import java.util.Arrays;
+import java.util.StringTokenizer;
 
+public class StringManipulation implements StringManipulationInterface {
+    String string;
     @Override
     public String getString() {
         return null;
     }
 
     @Override
     public void setString(String string) {
+        this.string = string;
     }
 
     @Override
     public int count() {
-        return 0;
```

Comment:
Also here, you forgot to delete as your pasted your code into the skeleton code.

Reply:
The - before the line indicates that it was removed from the last commit